piezo_web_app/PiezoWebApp/tests/services/spark_job/spark_job_service_test.py

```diff
@@ -200,3 +200,38 @@ def test_submit_job_logs_and_returns_api_exception_reason(self):
             'status': 999,
             'message': expected_message
         })
+
+    def test_get_job_status_sends_expected_arguments(self):
+        # Arrange
+        self.mock_kubernetes_adapter.get_namespaced_custom_object.return_value = {
+            'status': {
+                'applicationState': {
+                    'state': 'Running'}}}
```

Comment:
I'm surprised flake8 hasn't complained about the unusual indentation & imbalance of brackets here

Reply:
I'm surprised too but it complained until I had it like this